frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/top-consumers-card.tsx

```diff
@@ -94,4 +138,12 @@ const TopConsumerCard: React.FC<DashboardItemProps> = ({
   );
 };
 
-export default withDashboardResources(TopConsumerCard);
+export default connectToFlags(
+  ...getFlagsForExtensions(plugins.registry.getDashboardsStorageTopConsumerUsed()),
+)(withDashboardResources(TopConsumerCard));
+
+type topConsumersType = {
```

Comment:
i think adding `Type` to name is a bit redundant, maybe call it
```suggestion
type TopConsumersQueries = {
```

(with capital T)

Reply:
ack